run/test/specs/group_message_long_text.spec.ts

```diff
@@ -86,7 +85,9 @@ async function sendLongMessageGroupAndroid(platform: SupportedPlatformsType) {
       text: longText,
     }),
   ]);
-  await device2.replyToMessage(userA, longText);
+  await device2.longPressMessage(longText);
+  await device2.clickOnByAccessibilityID('Reply to message');
+  const replyMessage = await device2.sendMessage(`${userA.userName} message reply`);
```

Comment:
so I separated it out because the long text reply was replying with another massive long text instead of just 'Alice reply', and it was breaking android (their scroll to bottom button is a known issue)